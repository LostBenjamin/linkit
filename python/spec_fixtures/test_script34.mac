
[app 10]
    [[[sched 6 50]]]

(((object 6)))

[[object INSTANCE
  [sched-INSTANCE]
    INSTANCE
    red
]]

