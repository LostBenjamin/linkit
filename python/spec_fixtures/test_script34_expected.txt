10:set:11:run
4:set:0:red
5:set:1:red
6:set:2:red
7:set:3:red
8:set:4:red
9:set:5:red
11:set:50,4:sch:50,5:sch:50,6:sch:50,7:sch:50,8:sch:50,9:sch

